tv_apps/smart-system/js/app_install_manager.js

```diff
@@ -292,6 +322,72 @@ var AppInstallManager = {
     app.onprogress = this.handleProgress;
   },
 
+  previewApp: function ai_previewApp(app) {
+    if (!this.isMarketPlaceAppActive()) {
+      return;
+    }
+
+    var marketplace = AppWindowManager.getActiveApp();
+    var manifestURL = app.manifestURL;
+    var appURL = app.origin +
+      (app.manifest.launch_path === '/' ? '/' : app.manifest.launch_path);
```

Comment:
I don't really understand this line. Can't we simply use `appURL = app.origin + app.manifest.launch_path`?